publish_leap153.config

```diff
@@ -10,7 +10,7 @@ flavors=(DVD-x86_64 NET-x86_64 DVD-aarch64 NET-aarch64 DVD-ppc64le NET-ppc64le D
 get_version() {
 	# get expected version from first flavor
 	if [ -z "$version" ]; then
-		version=`echo $stage/iso/openSUSE-Leap-$leap_version-$flavor-Build???.?-Media.iso`
+		version=`echo $stage/iso/openSUSE-Leap-$leap_version-$flavor-Build??.?-Media.iso`
```

Comment:
some globe like    [0123456789]*.[0123456789]*            would make it more robust IMHO